cmd/release/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"time"
+)
+
+var (
+	targets = map[string][]string{
+		"linux": []string{
+			"amd64",
+			"386",
+		},
+		"darwin": []string{
+			"amd64",
+		},
+	}
+)
+
+const (
+	gitRepoURL = "github.com/MeilleursAgents/terraform-provider-ansiblevault"
+)
+
+func main() {
+	var flagAccessToken string
+	flag.StringVar(&flagAccessToken, "access-token", "", "help message for flagname")
+	flag.Parse()
+
+	date := time.Now().Format("2006-02-01_03:04:05PM")
+	tag, err := executeCmd([]string{}, "git", "tag", "--sort=-taggerdate")
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	app := new(flagAccessToken)
+
+	log.Println("[GOLANG] Build")
+	buildAssets, err := build(date, tag)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	log.Println("[GITHUB] Upload")
+	err = app.PublishArtifactsOnTag(tag, buildAssets)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func contains(a []string, x string) bool {
+	for _, n := range a {
+		if x == n {
```

Comment:
You might use `strings.EqualFold` in order to make a case insensitive search rather than strict comparison

Reply:
Done d5c53b5